### DISCUSSION TITLE
GraphQL Audit Logs is missing data

### DISCUSSION THREAD

**Discussion by barak-sgh** [community/community#64761] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

Seems like querying GraphQL for audit logs yields lot less results then querying `https://api.github.com/orgs/<org>/audit-log`.
For example `org_credential_authorization.grant` does appear in the API, though is missing from GQL.
Is there a reason for it? Is there a way to grab the rest of the results?

Best regards,
Barak

**Comment by arafajs** [comment]
**Yes**, there is a reason why querying GraphQL for audit logs yields fewer results than querying the REST API. **GraphQL** is a query language for APIs, and it allows you to specify exactly the data you want to retrieve. This can be helpful in preventing over-fetching of data, but it can also mean that you may not get all of the data that is available.

The **REST API**, on the other hand, returns all of the data that is available for a given endpoint. This means that you may get more data than you need, but you are guaranteed to get all of the data.

**Reply by barak-sgh** [reply]
@arafajs , Thanks for the reply. 
I would assume the rest of the data should be available from GraphQL as well no? I've searched the documentation top to bottom and couldn't find any way to get `org_credential_authorization` events. Is there anything that explains how to get the rest of the data that appears on the API?
